lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme_handlers.c

```diff
@@ -1610,7 +1614,18 @@ status_code_e s1ap_mme_handle_ue_context_release_complete(
           " UE Context Release commplete: S1 context should have been cleared "
```

Comment:
Please remove the comment before this log message and update the log message
```suggestion
          " UE Context Release complete: clearing S1 context"
```